### PR TITLE
fix Native to work with Array

### LINKED ISSUE #1211: Array of Objects is not wrapped by Native
```ruby
x = `[{"a": "b"},{"a":"c"},{"a":"d"}]`
y = Native(x)

# this is work
puts Native(y[0])["a"]

# this is not
puts y[0]["a"]
```

cc @meh

### PR DESCRIPTION
**Pull request by fazibear**
fix #1211

### CODE REVIEW

File: stdlib/native.rb [hunk 1]
```diff
@@ -213,6 +213,10 @@ def Native(obj)
       nil
     elsif native?(obj)
       Native::Object.new(obj)
+    elsif obj.is_a?(Array)
```
Comment (elia) [inline stdlib/native.rb]: This is good but should be probably generalized to any bridged class (Array, Numeric, Proc, String, …)

I also have a slight concern about perf.

Comment (fazibear) [inline stdlib/native.rb]: `Array` and `Proc` not work properly, but `String`, `Float`, `Integer` work good.

Comment (meh) [inline stdlib/native.rb]: As a note, each bridged class has a `$$bridge` special property.

**Comment by elia** [comment]
@fazibear thanks! 

Forgot to ask for a changelog entry before merging… (no worries, I'll add it :smile:)

**Pull request merged**